docs/UserGuide.md

```diff
@@ -26,40 +26,56 @@ This user guide aims to help new users quickly familiarise themselves with HRMat
+ - See <a href="#faq">How do I download Java 11?</a> if Java is not updated or if the terminal does not recognize the `java` input. 
 
 5. Type `java -jar hrmate.jar` into the terminal and press "enter" on your keyboard.<br>
 An app similar to the one below should appear in a few seconds. The app is populated with some sample data for you to experiment with.<br>
 We have added some annotations in <span style="color:red">red</span> so that you can understand the app visually.
 ![Ui](images/Ui-annotated.png)
 
 6. You can try typing some commands into the command box and pressing "enter" to execute. We have listed down some commands for you to try. <br>
- - `help` : Opens the help window.
- - `list` : Lists all contacts.
- - `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the employee list
- - `add-leave 1 title/medical leave start/2023-11-11 end/2023-11-11` : Adds an leave entry for the first employee in the current employee list. Note that the default leave status is `PENDING`.
- - `delete 3` : Deletes the 3rd employee shown in the current employee list.
- - `clear` : Deletes all employees and leaves.
- - `exit` : Exits the app.
+ - `help`: Opens the help window.
+ - `list`: Lists all employees.
+ - `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`: Adds an employee named `John Doe` to the employee list
+ - `add-leave 1 title/medical leave start/2023-11-11 end/2023-11-11`: Adds a leave entry for the first employee in the current employee list. Note that the default leave status is `PENDING`.
+ - `delete 3`: Deletes the 3rd employee shown in the current employee list.
+ - `clear`: Deletes all employees and leaves.
+ - `exit`: Exits the app.
 
 6. Refer to [How to interpret command formats?](#how-to-interpret-command-formats) for more info on the command structure. 
-Also, look through [Features](#features) for more commands and the details of each command. We hope to aid you in your HR journey!
+Also, look through [Features](#features) for commands and the details of each command. If you would like a concise summary of the commands, please check out our [command summary](#command-summary). We hope to aid you in your HR journey!
 
+--------------------------------------------------------------------------------------------------------------------
+## Important things to note
+> [!WARNING]  
+> HRmate is coded to take in the exact commands stated in this user guide. Please do not change the spelling, remove or add symbols, adjust spaces, or change the letter case of the commands. Any changes to the commands, unless otherwise stated in this user guide, can lead to the program being unable to interpret the input. To ensure the best user experience, please follow the commands exactly as described in this user guide. Check out our [command summary](#command-summary) for a concise list of available commands and their formats.
```

Comment:
This part seems ambiguous; which parts of the command can the user change? I think it might also be good to add a link to the command format interpretation section here (maybe shift from point 6) so the user knows which parts they are allowed to modify.

Reply:
updated